common/blobstore/blob/blob.go

```diff
@@ -34,21 +34,46 @@ func NewBlob(body []byte, tags map[string]string) *Blob {
 	}
 }
 
-// DeepCopy returns a deep copy of input blob
-func DeepCopy(blob *Blob) *Blob {
-	if blob == nil {
+// DeepCopy returns a deep copy of blob
+func (b *Blob) DeepCopy() *Blob {
+	if b == nil {
```

Comment:
Looks like this nil check is not needed anymore.

Reply:
The receiver could still potentially be nil. It is a bug if this method is called anywhere with a nil receiver but it's possible.